bindings/ergo-lib-wasm/tests_browser/test_rest_api.js

```diff
@@ -14,13 +13,56 @@ beforeEach(async () => {
 it('node REST API: get_nipopow_proof_by_header_id endpoint', async () => {
     let node_conf = new ergo_wasm.NodeConf("213.239.193.208:9053");
     assert(node_conf != null);
-    const block_headers = generate_block_headers();
-    const header_id = block_headers.get(0).id();
+    const header_id = ergo_wasm.BlockId.from_str("4caa17e62fe66ba7bd69597afdc996ae35b1ff12e0ba90c22ff288a4de10e91b");
     let res = await ergo_wasm.get_nipopow_proof_by_header_id(node_conf, 3, 4, header_id);
     assert(res != null);
     assert(node_conf != null);
 });
 
+it('node REST API: example SPV workflow', async () => {
+    const header_id = ergo_wasm.BlockId.from_str("d1366f762e46b7885496aaab0c42ec2950b0422d48aec3b91f45d4d0cdeb41e5")
+    assert(header_id != null);
+    let tx_id = ergo_wasm.TxId.from_str("258ddfc09b94b8313bca724de44a0d74010cab26de379be845713cc129546b78");
+    assert(tx_id != null);
+
+    // Taken from: https://explorer.ergoplatform.com/en/blocks/d1366f762e46b7885496aaab0c42ec2950b0422d48aec3b91f45d4d0cdeb41e5
+    let transactions_root = ergo_wasm.base16_decode("be1e2428e9e8c932ff2bcbc9075537db36bb704b9cd7ae86e11219c66ba52c0e");
```

Comment:
In the spirit of keeping this example real let's fetch the header with tx root from the node as well.

Reply:
Done.